packaging/docker/build.sh

```diff
@@ -19,6 +19,11 @@ ARCH_MAP=(["i386"]="386" ["amd64"]="amd64" ["armhf"]="arm" ["aarch64"]="arm64")
 DEVEL_ARCHS=(amd64)
 [ "${ARCHS}" ] || ARCHS="${!ARCH_MAP[@]}" # Use default ARCHS unless ARCHS are externally provided
 
+if [ "$RELEASE_CHANNEL" != "nightly" ] && [ "$RELEASE_CHANNEL" != "stable" ]; then
```

Comment:
I think we should follow the convention of the rest of the script and use "${RELEASE_CHANNEL}" instead of "$RELEASE_CHANNEL"

```suggestion
if [ "${RELEASE_CHANNEL}" != "nightly" ] && [ "${RELEASE_CHANNEL}" != "stable" ]; then
```